tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -94,6 +95,8 @@ def convert_value(value, type_):
         elif value == 'false' or value == '0':
             return False
         raise ValueError("%s is not a '%s' type"%(value, type_))
+    elif type_ == 'yaml':
+        return yaml.load(value)
```

Comment:
Should any yaml specific exceptions be catched here and a `ValueError` be raised instead?